src/transformers/pipelines/token_classification.py

```diff
@@ -179,6 +189,30 @@ def _sanitize_parameters(
             postprocess_params["aggregation_strategy"] = aggregation_strategy
         if ignore_labels is not None:
             postprocess_params["ignore_labels"] = ignore_labels
+        if process_all is True:
+            if self.tokenizer.is_fast:
+                if stride is not None:
+                    if stride < 0:
+                        stride = 0
+                        warnings.warn(
+                            "`stride` cannot be a negative number, defaulted to" f" `stride={stride}` instead."
+                        )
+                else:
+                    stride = 0
+                tokenizer_params = {
+                    "return_overflowing_tokens": True,
+                    "max_length": self.tokenizer.model_max_length,
```

Comment:
Remove this.

In combination with padding it's a HUGE performance loss, potentially crashing things.
Some tokenizer do not have a max_length and this is set to basically infinity, using this with padding has potential to blow memory up.

If the tokenizer handles infinite lengths, there's no need to chunk anything.